specs.tex

```diff
@@ -7,18 +7,26 @@ \subsection{Arena}
 \label{spec:arena}
 
 \begin{enumerate}
-  \item The arena floor is a \SI{6000}{mm} $\times$ \SI{12000}{mm} rectangle.
+  \item The arena floor is an \SI{8.4}{m} $\times$ \SI{8.4}{m} rectangle. The
```

Comment:
```suggestion
  \item The arena floor is an \SI{5.4}{m} $\times$ \SI{5.4}{m} rectangle. The
```

Reply:
The reason for 5.4 was to match the SR wooden arena. What are we using for this to build the arena?

Reply:
No, the reason for the original 8.4 was based on truss lengths. The new 5.4m is based on the available truss.